.github/workflows/build-and-run.yml

```diff
@@ -215,3 +216,29 @@ jobs:
         run: sleep 1
       - name: Check accel-ppp stat
         run: accel-cmd show stat
+
+  Build-in-Container-Alpine:
+    runs-on: ubuntu-latest
+    container:
+      image: alpine:3.19
+    steps:
+      - name: Install build tools
+        run: >
+          apk update && apk add --no-cache git cmake make g++ pcre-dev libressl-dev linux-headers libucontext-dev lua5.1-dev
+      - name: Check out repository code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: mkdir build
+        run: mkdir build
+      - name: Disable git security warnings
+        run: git config --global --add safe.directory '*'
+      - name: cmake
+        working-directory: ./build
+        run: >
+          cmake -DBUILD_IPOE_DRIVER=FALSE -DBUILD_VLAN_MON_DRIVER=FALSE -DCMAKE_INSTALL_PREFIX=/usr
+          -DKDIR=/usr/src/linux-headers-`uname -r`
+          -DLUA=TRUE -DSHAPER=FALSE -DRADIUS=TRUE ..
+      - name: make
+        working-directory: ./build
+        run: make
```

Comment:
Let's run accel-ppp with default config and check 'show stat' command like it's done for debian-based containers

Reply:
done